### PR TITLE
translatable classProviderRef

### PR DESCRIPTION
**Pull request by njr-11**
Translatable name and description for classProviderRef.
This is kept separate from the other changes so it does not hold up delivery if we go back and forth several times on the language to be used.

### CODE REVIEW

File: dev/com.ibm.ws.security.jaas.common/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -35,6 +35,10 @@ jaasLoginModule.desc=A login module in the JAAS configuration.
 className=Class name
 className.desc=Fully-qualified package name of the JAAS login module class.
 
+classProviderRef=Class Provider
+classProviderRef.desc=A reference to the ID of the resource adapter or application that provides the JAAS login module class. Only specify this configuration attribute if you do not specify a shared library reference.
```
Comment (dmuelle) [inline dev/com.ibm.ws.security.jaas.common/resources/OSGI-INF/l10n/metatype.properties]: **change**
Only specify this configuration attribute if you do not specify a shared library reference.

**to**
Specify this configuration attribute only if you do not specify a shared library reference.

Comment (njr-11) [reply dev/com.ibm.ws.security.jaas.common/resources/OSGI-INF/l10n/metatype.properties]: Thanks - I added commit 
https://github.com/OpenLiberty/open-liberty/pull/10885/commits/b72e9c77050c1b12b9d858e0901e3b0f0462ba58
with the requested change.  Please approve the review if I've copied it over correctly.

**Comment by njr-11** [comment]
#build
#run-libby-bot

**Comment by skasund** [comment]
L2 message review no required

**Pull request merged**